linera-chain/src/chain.rs

```diff
@@ -602,14 +645,37 @@ where
         }
         let origins = events_by_origin.keys().copied();
         let inboxes = self.inboxes.try_load_entries_mut(origins).await?;
+        let mut removed_unskippable = HashSet::new();
         for ((origin, events), mut inbox) in events_by_origin.into_iter().zip(inboxes) {
             tracing::trace!("Updating inbox {:?} in chain {:?}", origin, chain_id);
             for event in events {
                 // Mark the message as processed in the inbox.
-                inbox
+                let was_added = inbox
```

Comment:
Question: should this be called `was_removed` instead? Or maybe the goal was a meaning like `was_present`?